rfc6962-bis.xml

```diff
@@ -759,14 +826,14 @@ messages.
             <t hangText="Inputs:">
               <list style="hanging">
                 <t hangText="chain:">
-                  An array of base64 encoded certificates. The first element is the end-entity certificate; the second chains to the first and so on to the last, which is either the root certificate or a certificate that chains to a known root certificate.
+                  An array of base64 encoded certificates. The first element is the certificate for which the submitter desires an SCT; the second chains to the first and so on to the last, which is either an accepted root certificate or a certificate that chains to an accepted root certificate.
```

Comment:
Let's do that in http://trac.tools.ietf.org/wg/trans/trac/ticket/102, as well as do "s/accepted root certificate/accepted trust anchor/".  I think we should add an "Accepted Trust Anchors" section just before the "Log Entries" section.